ProductDetails/src/components/Products.jsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable camelcase */
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+import axios from 'axios';
+// eslint-disable-next-line import/extensions
+import ImageGallery from './LeftSection/ImageGallery.jsx';
+// eslint-disable-next-line import/extensions
+import ProductDetails from './RightSection/ProductDetails.jsx';
+// eslint-disable-next-line import/extensions
+import ProductInfo from './ProductInfo.jsx';
+
+class Products extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      product: {},
+      styles: [],
+    };
+    this.getStyles = this.getStyles.bind(this);
+  }
+
+  // eslint-disable-next-line react/sort-comp
+  getStyles(styles) {
+    // eslint-disable-next-line react/no-unused-state
+    this.setState({ styles });
+  }
+
+  componentDidMount() {
+    const id = 14807;
+    axios.get(`/products/${id}`)
+      .then((response) => {
+        const { data } = response;
+        const {
+          category, default_price, description, features, name, slogan,
+        } = data;
+        // eslint-disable-next-line object-curly-newline
+        // eslint-disable-next-line max-len
+        this.setState({ product: { category, default_price, description, features, name, slogan } });
```

Comment:
ESLint is insane. They want this: 
```suggestion
                this.setState({
          product: {
            category, default_price, description, features, name, slogan,
          },
        });
      })
```